stdlib/md_renderer.rs

```diff
@@ -69,16 +77,8 @@ impl Renderer for MarkdownRenderer {
 // HELPER FUNCTIONS
 // ================================================================================================
 
-fn get_module_name(ns: &str) -> String {
-    let parts: Vec<&str> = ns.split(LibraryPath::PATH_DELIM).collect();
-    String::from(parts[parts.len() - 1])
-}
-
-fn get_module_section(ns: &str) -> String {
-    let parts: Vec<&str> = ns.split(LibraryPath::PATH_DELIM).collect();
-    String::from(parts[parts.len() - 2])
-}
-
-fn markdown_file_name(ns: &str) -> String {
-    format!("{}_{}.md", get_module_name(ns), get_module_section(ns))
+fn get_dir_and_file(ns: &str) -> (Vec<&str>, &str) {
+    let ns_parts: Vec<&str> = ns.split(LibraryPath::PATH_DELIM).collect();
+    let (dir_parts, file_name) = ns_parts.split_at(ns_parts.len() - 1);
+    (dir_parts.to_vec(), file_name[0])
```

Comment:
Why not use a simple `pop()` to get the file name out of `ns_parts` here?

Also, should we handle logic of lines 54 - 56 here? Basically, return a `PathBuf` and a string with full file path from here.